alerts/cluster-network-operator/NoOvnClusterManagerLeader.md

```diff
@@ -0,0 +1,81 @@
+# NoOvnClusterManagerLeader
+
+## Meaning
+
+This [alert][NoOvnClusterManagerLeader] is triggered when ovn-kubernetes
+cluster does not have a leader for more than 10 minutes.
+
+## Impact
+
+When ovnkube cluster manager is unable to elect a leader (via kubernetes lease
+API), networking control plane is degraded.
+Networking configuration updates applied to the cluster will not be
+implemented while there is no OVN Kubernetes cluster manager leader.
+Existing workloads should continue to have connectivity.
+OVN-Kubernetes control plane is not functional.
+
+## Diagnosis
+
+### Fix alerts before continuing
+
+Check to ensure the following alerts are not firing and resolved before
+continuing as they may cause this alert to fire:
+
+[Alert hierarchy](./hierarchy/alerts-hierarchy.svg)
+
+### OVN-kubernetes control-plane pods
+
+Check if all the ovn-kube control planes are running:
```

Comment:
Can you remove this? Cause if theres no CM leader, its unlikely they control planes are running anyway and theyll be crashlooping.